### DISCUSSION TITLE
Setting Member Privilege for team creation in orgs via API

### DISCUSSION THREAD

**Discussion by autumnfound** [community/community#24712] [Apps, API and Webhooks]
<p>Hey, I’ve looked through a few parts of the docs, but it looks like the setting to restrict Member Team Privileges via the orgs endpoint in the REST API. There are other internal member privileges that are supported, but not this one. There doesn’t appear to be a preview to enable it either, which I’ve used a few times in the past. Is this a gap that is planned to be covered in the near future, or is this setting intended to only be set manually via the UI?</p>
<p>Setting in question: <a href="https://docs.github.com/en/github/setting-up-and-managing-organizations-and-teams/setting-team-creation-permissions-in-your-organization" class="inline-onebox" rel="noopener nofollow ugc">Setting team creation permissions in your organization - GitHub Docs</a></p>
<p>Current org endpoint def: <a href="https://docs.github.com/en/rest/reference/orgs#update-an-organization" class="inline-onebox" rel="noopener nofollow ugc">Organizations - GitHub Docs</a></p>

**Comment by nethgato** [comment, marked as answer]
<p>Hey <a class="mention" href="/autumnfound">@autumnfound</a> :wave:</p>
<p>Thanks so much for your post and for joining the Community!</p>
<p>I do see what you mean. Currently, there is no way to toggle the allow/disallow Teams creation for Org members via the API.</p>
<p>I also know that over on the Enterprise side, a number of customers have requested similar, and have been added to an internal request thread. If you are a member of a GHE/S Organization, I might recommend opening a support ticket with them, directly:</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <a href="https://enterprise.github.com/support" target="_blank" rel="noopener">enterprise.github.com</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/362;"><img src="https://user-images.githubusercontent.com/62281706/181086390-c1ed95a6-64bf-4882-844a-2fdfe08f884f.png" class="thumbnail" width="690" height="362" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/d/d906e4c1a04a608e211bb6dba332357009de24be_2_690x362.png, https://aws1.discourse-cdn.com/github/optimized/2X/d/d906e4c1a04a608e211bb6dba332357009de24be_2_1035x543.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/d/d906e4c1a04a608e211bb6dba332357009de24be.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/d/d906e4c1a04a608e211bb6dba332357009de24be_2_10x10.png"></div>

<h3><a href="https://enterprise.github.com/support" target="_blank" rel="noopener">Support - GitHub Enterprise</a></h3>

<p>The power of GitHub's social coding for your own workgroup. Pricing, tour and more.</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>If you’re a non-Enterprise user, I highly recommend submitting via this feedback form:</p>
<aside class="onebox githubfolder">
  <header class="source">
      <img src="https://support-assets.github.com/assets/favicon-3dcd2c977a91b2e3f396b300c5ff0d572295ca632ddb7ba1adcacc1ea68dd5c0.ico" class="site-icon" width="32" height="32">
      <a href="https://support.github.com/contact/feedback" target="_blank" rel="noopener">support.github.com</a>
  </header>
  <article class="onebox-body">
    

<h3><a href="https://support.github.com/contact/feedback" target="_blank" rel="noopener">Share feedback - GitHub Support</a></h3>

<p><a href="https://support.github.com/contact/feedback" target="_blank" rel="noopener">//support.github.com/contact/feedback</a></p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>…to add your +1 for this to be added to either REST or GraphQL, or both!</p>
<p>Thanks again for bringing this up. :bow:</p>

**Comment by autumnfound** [comment]
<p>Thanks for getting back (and sorry for late response lol!), I know we have licensing/product arrangements w/ Github, but I don’t know if we qualify as Enterprise. I’ll let my team know and we’ll hit on the solution given.</p>
<p>I’ve marked your answer as solution as there is no current endpoint for this. I’d probably give my +1 to REST as that’s what we use internally for our binding!</p>

**Comment by autumnfound** [comment]
<p>I’ve been asked by my team to follow up and I figure this is the least intrusive way to do so! We’ve been getting some pushes in the Eclipse community about changing groups to “public” from “secret”, and managing this setting is the only blocker to this currently (<a href="https://bugs.eclipse.org/bugs/show_bug.cgi?id=574879#c20" class="inline-onebox" rel="noopener nofollow ugc">574879 – GitHub Mirror for Scout removed?</a>). Is there any public resource I can follow on this issue in the future to point them at to reduce pings and other such things? Thanks in advance!</p>
